privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -225,28 +221,10 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         this.site.value = site
     }
 
-    private fun pixelParamList(): List<String> {
-        val viewState = viewState.value ?: return emptyList()
-        if (viewState.remoteFeatureSettings.primaryScreen.layout == LayoutType.DEFAULT.value) return emptyList()
-        // otherwise, send the pixel param
-        return listOf(DASHBOARD_TOGGLE_HIGHLIGHT.pixelName)
-    }
-
-    private fun pixelParamMap(): Map<String, String> {
-        return pixelParamList().associateWith { true.toString() }
-    }
-
     private suspend fun createRemoteFeatureSettingsViewState() = withContext(dispatcher.io()) {
-        val altLayoutEnabled = privacyDashboardRemoteFeature.highlightedProtectionsToggle().isEnabled()
-        val isEnglish = Locale.getDefault().language == Locale.ENGLISH.language
-        val primaryLayout = if (altLayoutEnabled && isEnglish) {
-            LayoutType.HIGHLIGHTED_PROTECTIONS_TOGGLE.value
-        } else {
-            LayoutType.DEFAULT.value
-        }
         return@withContext RemoteFeatureSettingsViewState(
             primaryScreen = PrimaryScreenSettings(
-                layout = primaryLayout,
+                layout = LayoutType.DEFAULT.value,
```

Comment:
Don't need to pass any values here since the default is already the one your are passing. Also, can we just remove this? It's always the same value so I don't think it makes sense to pass anything at all.